src/modm/board/nucleo_f042k6/board.hpp

```diff
@@ -55,7 +55,11 @@ struct SystemClock {
 	{
 		Rcc::enableInternalClock();	// 8MHz
 		// (internal clock / 2) * 12 = 48MHz
-		Rcc::enablePll(Rcc::PllSource::InternalClock, 12, 2, 1);
+		const Rcc::PllFactors pllFactors{
+			.pllMul = 12,
+			.pllPrediv = 1
+		};
+		Rcc::enablePll(Rcc::PllSource::InternalClock, pllFactors);
```

Comment:
Fairly certain this was broken before and is correct now. See NUCLEO-F031 and F051-DISCO for comparison.

Reply:
But please set the pll pre-divider to 2:
```diff
+ .pllPrediv = 2
- .pllPrediv = 1
```

Reply:
![grafik](https://user-images.githubusercontent.com/2820734/77225442-516f1e00-6b6f-11ea-940e-46a61bc17a5b.png)

Reply:
Use /1 *6.